_data/simple-icons.json

```diff
@@ -5665,12 +5670,14 @@
         {
             "title": "Octopus Deploy",
             "hex": "2F93E0",
-            "source": "https://octopus.com/company/brand"
+            "source": "https://octopus.com/company/brand",
+            "guidelines": "https://octopus.com/company/brand"
```

Comment:
Is the 'guidelines' link necessary if it's the same location as the source? My understanding was it was only necessary if explicit guidelines regarding the usage of the icon were on a different page to where the icon was sourced. There's a fair few which simply seem to duplicate the existing source URL. I won't review every single instance, but we may be able to remove those.

Reply:
> The guidelines may be used to specify the URL of the brand's guidelines/press kit/etc. This is useful if the SVG file was sourced from a different place, still if the SVG file was sourced from the guidelines, the URL should be duplicated here.

According to the current [guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md)

Reply:
That feels like unnecessary duplication on our part - though I guess if there are explicit guidelines and not just 'use this icon' we should include them. What do other @simple-icons/maintainers feel on this?